coursera_autograder/commands/update_resource_limits.py

```diff
@@ -37,15 +37,11 @@ def command_update_resource_limits(args):
     s.auth = auth
 
     course_branch_id = (args.course.replace("~", "!~")
-                        if "authoringBranch" in args.course else args.course)
+                        if "authoringBranch~" in args.course else args.course)
     course_branch_item = '%s~%s' % (course_branch_id, args.item)
 
     params = 'id=%s&partId=%s' % (course_branch_item, args.part)
 
-    if args.grader_cpu is not None and args.grader_cpu not in {'1', '2', '4'}:
```

Comment:
Why are we are removing this?

Reply:
This check was supposed to be handled by the parser -- I must have made the change in a different commit somewhere.